src/CellsTexturesManager.cpp

```diff
@@ -107,7 +106,7 @@ sf::Texture& CellsTexturesManager::getTextureReferenceByCellType(
     }
     else
     {
-        return wallCellTexture;
+        return emptyCellTexture;
```

Comment:
add comments for each case, explanation of the default one and why this one is default